emscripten.py

```diff
@@ -1669,7 +1665,7 @@ def create_the_global(metadata):
   if metadata['simd'] or shared.Settings.SIMD:
     # Always import SIMD when building with -s SIMD=1, since in that mode memcpy is SIMD optimized.
     fundamentals += ['SIMD']
-  return '{ ' + ', '.join(['"' + math_fix(s) + '": ' + s for s in fundamentals]) + ' }'
```

Comment:
It looks like the `fundamentals` list here never includes anything that fix_importa_name can effect so we can probably drop it?  (I.e. non of the fundamentals start with `_` or `Math_`.

Reply:
got it